otter_welcome_buddy/cogs/english_club_reminder.py

```diff
@@ -0,0 +1,139 @@
+
+
+class English(commands.Cog):
+    """
+    English command events, where notifications about English sessions
+    Commands:
+        englishclub start:               Start cronjob for english club reminders messages
+        englishclub stop:                Stop cronjob for english club reminders messages
+
+    schedule_session:    Allows us to schedule an English session
+    """
+
+    def __init__(self, bot: Bot, messages_dependency: type[messages.Formatter]):
+        self.bot: Bot = bot
+        self.messages_formatter: type[messages.Formatter] = messages_dependency
+        self.scheduler: AsyncIOScheduler = AsyncIOScheduler()
+        self.__configure_scheduler()
+
+    @commands.group(
+        brief="Commands related to English Club!",
+        invoke_without_command=True,
+        pass_context=True,
+    )
+    @commands.has_any_role(OTTER_ADMIN, OTTER_MODERATOR)
+    async def englishclub(self, ctx: Context) -> None:
+        """Displays englishclub commands"""
+        await ctx.send_help(ctx.command)
+
+    @englishclub.command(brief="Start the cronjob for the messages")  # type: ignore
+    @commands.has_any_role(OTTER_ADMIN, OTTER_MODERATOR)
+    async def start(self, _: Context) -> None:
+        """Command to interact with the bot and start cron"""
+
+        self.__configure_scheduler()
+
+    @englishclub.command(brief="Stop the cronjob for the messages")  # type: ignore
+    @commands.has_any_role(OTTER_ADMIN, OTTER_MODERATOR)
+    async def stop(self, _: Context) -> None:
+        """Command to interact with the bot and stop cron"""
+
+        self.scheduler.remove_job(ENGLISH_CLUB_JOB_ID)
+
+    @commands.command(brief="Schedule an English session")
+    @commands.has_any_role(OTTER_ADMIN, OTTER_MODERATOR)
+    async def schedule_session(self, ctx: Context, *hour: str) -> None:
+        """Schedules the English Club session"""
+
+        # Regex pattern explanation:
+        # ^: Asserts the start of the string.
+        # (1[0-2]|0?[1-9]): Matches an hour with the 12-hour format.
+        # : Matches the colon that separates the hour and minute.
+        # ([0-5][0-9]): Matches exactly 2 digits for the minutes, from "00" to "59".
+        #  ?: Matches zero or one space, allowing for space between the minutes and AM/PM.
+        # (AM|PM): Matches either "AM" or "PM" (case-sensitive).
+        # $: Asserts the end of the string.
+        time_format_regex = r"^(1[0-2]|0?[1-9]):([0-5][0-9]) ?(AM|PM)$"
+        hour_str: str = " ".join(hour)
+
+        if re.match(time_format_regex, hour_str):
+            channel_id: int = ENGLISH_CHANNEL_ID
+            channel: DiscordChannelType | None = get_channel_by_id(self.bot, channel_id)
+            everyone = "@everyone"
+
+            embed = discord.Embed(
+                colour=discord.Colour.green(),
+                title="English Club Session",
+                description=everyone + self.messages_formatter.english_club_message(hour_str),
```

Comment:
Try to be consistent about the usage of `f strings` and `+` to concatenate strings.

Reply:
changes has been made